app/routes/session.py

```diff
@@ -178,7 +179,21 @@ def get_sign_out():
 
 @session_blueprint.route("/signed-out", methods=["GET"])
 def get_signed_out():
-    return render_template(template="signed-out")
+    if not cookie_session:
+        return redirect(url_for("session.session_expiry"))
```

Comment:
This should go to session expired not expiry.
```suggestion
        return redirect(url_for("session.get_session_expired"))
```